src/Microsoft.AspNetCore.Authentication.Abstractions/AuthenticateResult.cs

```diff
@@ -64,22 +58,12 @@ public static AuthenticateResult Success(AuthenticationTicket ticket)
         }
 
         /// <summary>
-        /// Indicates that stage of authentication was directly handled by user intervention and no
-        /// further processing should be attempted.
-        /// </summary>
-        /// <returns>The result.</returns>
-        public static AuthenticateResult Handle()
-        {
-            return new AuthenticateResult() { Handled = true };
-        }
-
-        /// <summary>
         /// Indicates that there was no information returned for this authentication scheme.
         /// </summary>
         /// <returns>The result.</returns>
-        public static AuthenticateResult None()
+        public static AuthenticateResult NoResult()
```

Comment:
That would leave you with a method and property that have the same name. NoResult is fine here.